### ISSUE TITLE
Does not work with 'rtsp streams!

### ISSUE THREAD

**Issue by zazeblik** [moust/cordova-plugin-videoplayer#24]
In VLC I write this URL rtsp://<login>:<password>@172.17.2.109:554 and everything appears fine, but when I write in my code
document.addEventListener('deviceready', function() { 
VideoPlayer.play(
    "rtsp://<login>:<password>@172.17.2.109:554",
    {
        volume: 0.5
    },
    function () {
        alert("video completed");
    },
    function (err) {
        alert(err);
    }
);
...
}
alert говорит мне "Prepare failed: status=0x1". Работает ли плагин с RTSP?

**Comment by dfromero** [comment]
hi @zazeblik did you find any workaround/solutions, I'm facing the same issue.

**Comment by zazeblik** [comment]
No. At the moment, I send MJPEG stream from the camera directly into the IMG tag like this : <img src="http://--login--:--password--@172.17.2.85/ipcam/mjpeg.cgi"></img> But you have to sacrifice sound ...

**Comment by Zorgatone** [comment]
Any update on this? @zazeblik, @dfromero guys?

I need support for authenticated RTSP streams on iOS and Android. Did you find any solution for this?

@moust Can that be done, or is it a limitation of the OS native player?

**Comment by zazeblik** [comment]
No

**Comment by moust** [comment]
I have no idea, I'm not mobile developer. Maybe some one is more advances knowledges with Android development will be able to help you.

**Comment by alieslamifard** [comment]
there are many plugin for supporting rtsp, hls, dash and another stream method.
use theme instead.
jwplayer, clappr, ...

**Comment by mburger81** [comment]
@alieslamifard, it is a pain finding a rtsp supported player.  clappr and jwplayer should not have rtsp support, or? 
@zazeblik did you found a solution for this or another player / plugin?

**Comment by zazeblik** [comment]
Not20 авг. 2017 г. 12:04 AM пользователь mburger81 <notifications@github.com> написал:@alieslamifard, it is a pain finding a rtsp supported player.  clappr and jwplayer should not have rtsp support, or?
@zazeblik did you found a solution for this or another player / plugin?

**Comment by dfromero** [comment]
I remember somehow fixing this issue, not sure with which plugin, but it was a cordova official plugin or this one
